src/main/java/com/fasterxml/jackson/databind/deser/impl/PropertyValueBuffer.java

```diff
@@ -176,6 +179,34 @@ public Object[] getParameters(SettableBeanProperty[] props)
                 }
             }
         }
+
+        // [databind#562]: Respect @JsonAnySetter in @JsonCreator
+        // check if we have anySetter Param
+        AnnotationIntrospector ai = _context.getAnnotationIntrospector();
+        for (int i = 0; i < props.length; i++) {
+            SettableBeanProperty prop = props[i];
+            AnnotatedMember member = prop.getMember();
+            if (member == null) {
+                continue;
+            }
+            Boolean hasAnySetter = ai.hasAnySetter(member);
+            if (hasAnySetter == null || !hasAnySetter) {
+                continue;
+            }
```

Comment:
Same as '_findCreatorPropWithAnySetter()' method above. Wonder if we can add this method to CreatorProperty class?